readme.txt

```diff
@@ -8,58 +8,80 @@ License: GPLv2 or later
 
 == Frequently Asked Questions ==
 
-Please see the [FAQs on amp-wp.org](https://amp-wp.org/documentation/frequently-asked-questions/). Don't see an answer to your question? Please [search the support forum](https://wordpress.org/support/plugin/amp/) to see if someone has asked your question. Otherwise, please [open a new support topic](https://wordpress.org/support/plugin/amp/#new-post).
+Please see the [FAQs on amp-wp.org](https://amp-wp.org/documentation/frequently-asked-questions/). Don&#39;t see an answer to your question? Please [search the support forum](https://wordpress.org/support/plugin/amp/) to see if someone has asked your question. Otherwise, please [open a new support topic](https://wordpress.org/support/plugin/amp/#new-post).
 
 == Installation ==
 
 1. Upload the folder to the `/wp-content/plugins/` directory.
-2. Activate the plugin through the 'Plugins' menu in WordPress.
+2. Activate the plugin through the &#39;Plugins&#39; menu in WordPress.
 3. If you currently use older versions of the plugin in `Reader` mode, it is strongly encouraged to migrate to `Transitional` or `Standard` mode. Depending on your theme/plugins, some development work may be required.
 
 == Getting Started ==
 
-To learn more about the plugin and start leveraging its capabilities to power your AMP content creation workflow check [the official AMP plugin product site](https://amp-wp.org).
+To learn more about the plugin and start leveraging its capabilities to power your AMP content creation workflow check [The Official AMP Plugin product site](https://amp-wp.org/).
 
 If you are a developer, we encourage you to [follow along](https://github.com/ampproject/amp-wp) or [contribute](https://github.com/ampproject/amp-wp/blob/develop/contributing.md) to the development of this plugin on GitHub.
 
+We have put up a comprehensive [FAQ page](https://amp-wp.org/documentation/frequently-asked-questions/) and extensive documentation to help you start as smoothly as possible.
+
+But if you need some help, we are right here to support you on this plugin&#39;s forum section, as well as through Github issues. And yep, our thriving [AMPExpert ecosystem](https://amp-wp.org/ecosystem/amp-experts/) has indie freelancers to enterprise grade agencies in case you need commercial support!
```

Comment:
```suggestion
But if you need some help, we are right here to support you in the plugin's [support forum](https://wordpress.org/support/plugin/amp/), as well as through [GitHub issues](https://github.com/ampproject/amp-wp/issues) (for technical bugs and feature requests). And yep, our thriving [AMP Expert ecosystem](https://amp-wp.org/ecosystem/amp-experts/) has indie freelancers to enterprise grade agencies in case you need commercial support!
```